src/Groups/perm.jl

```diff
@@ -772,15 +779,36 @@ end
 
 ################################################################################
 #
-#   perm
+#   @perm
 #
+
 @doc raw"""
-    @perm ex
-    
-Input a permutation in cycle notation. Supports arbitrary expressions for
-generating the integer entries of the cycles. The parent group is inferred 
-to be the symmetric group with a degree of the highest integer referenced 
-in the permutation.
+    @perm expr
+    @perm n expr
+    @perm G expr
+
+Input a permutation or a non-empty list of permutations in cycle notation.
+
+Supports arbitrary expressions for generating the integer entries of the cycles.
+
+`expr` may either be a single permutation, a non-empty list of permutations,
+or a non-empty tuple of permutations. **Note:** `@perm ()` denotes the identity
+permutation, NOT the empty tuple of permutations.
+
+If a group `G` is provided, the permutations are created as elements of `G`. This will
+raise an error if the permutations are not elements of `G`.
+
+If an integer `n` is provided, the permutations are created as elements of the
+symmetric group of degree `n`, i.e., `symmetric_group(n)`.
+
+Input a list of permutations in cycle notation, created as elements of the
+symmetric group of degree `n`, i.e., `symmetric_group(n)`, by invoking
+[`cperm`](@ref) suitably.
```

Comment:
I am confused by this paragraph. Is this meant to suggest an alternative? Then make that explicit:
```suggestion
As an alternative to the `@perm` macro, the [`cperm`](@ref) function
also allows creating permutations as a product of cycles.
```

I would also move it to the end?

Reply:
I just see that the example now also shows `cperm` and `perm`. OK, then perhaps this text should also mention (and ref) `perm` as an alternative to `@perm`?

Reply:
This was left from the previous docstring. I removed that and instead added a mention to `perm` and `cperm`.